node/src/res/qf-local.plain.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "Local Testnet",
+  "id": "local_testnet",
+  "chainType": "Local",
+  "bootNodes": [
+    "/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWBAtaXQXjKZrrZHR57fa8N1zyMfXXfPLUZdw31iKGEPWT"
```

Comment:
Remove? Zombienet nodes will have a new identity each time, this will produce an error in logs.

Reply:
Removed